### PR TITLE
Update dependencies for release

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

**Comment by fredj** [comment]
LGTM

**Comment by gberaudo** [comment]
Thanks @fredj. Added one more commit for switching to Eslint. Please review.

File: README.md [hunk 1]
```diff
@@ -56,7 +56,6 @@ Requirements for building OL3-Cesium:
 * [GNU Make](http://www.gnu.org/software/make/)
 * [Node.js](http://nodejs.org/)
 * [Python](http://python.org/)
```
Comment (fredj) [inline README.md]: Python can also be removed

Comment (gberaudo) [inline README.md]: Thanks, done

**Comment by fredj** [comment]
LGTM

**Pull request merged**